applications/GeoMechanicsApplication/tests/test_conditions/test_UPwNormalFaceLoadCondition3D3N/ProjectParameters.json

```
@@ -0,0 +1,159 @@
+{
+    "problem_data": {
+        "problem_name":         "test_UPwNormalFaceLoadCondition3D3N",
+        "start_time":           0.0,
+        "end_time":             1200.0,
+        "echo_level":           1,
+        "parallel_type":        "OpenMP",
+        "number_of_threads":    1
+    },
+    "solver_settings": {
+        "solver_type":                        "U_Pw",
+        "model_part_name":                    "PorousDomain",
+        "domain_size":                        3,
+        "model_import_settings":              {
+            "input_type":       "mdpa",
+            "input_filename":   "test_UPwNormalFaceLoadCondition3D3N"
+        },
+        "material_import_settings":              {
+            "materials_filename":       "../Common/MaterialParametersUPw3D.json"
+        },
+        "time_stepping":              {
+            "time_step":                600,
+            "max_delta_time_factor":    1
+        },
+        "buffer_size":                        2,
+        "echo_level":                         1,
+        "clear_storage":                      false,
+        "compute_reactions":                  true,
+        "move_mesh_flag":                     false,
+        "reform_dofs_at_each_step":           false,
+        "nodal_smoothing":                    false,
+        "block_builder":                      true,
+        "solution_type":                      "Quasi-Static",
+        "scheme_type":                        "Backward_Euler",
+        "reset_displacements":                true,
+        "newmark_beta":                       0.25,
+        "newmark_gamma":                      0.5,
+        "newmark_theta":                      0.5,
+        "rayleigh_m":                         0.0,
+        "rayleigh_k":                         0.0,
+        "strategy_type":                      "newton_raphson",
+        "convergence_criterion":              "displacement_criterion",
+        "displacement_relative_tolerance":    1.0E-4,
+        "displacement_absolute_tolerance":    1.0E-9,
+        "residual_relative_tolerance":        1.0E-4,
+        "residual_absolute_tolerance":        1.0E-9,
+        "water_pressure_relative_tolerance":  1.0E-4,
+        "water_pressure_absolute_tolerance":  1.0E-9,
+        "min_iterations":                     6,
+        "max_iterations":                     15,
+        "number_cycles":                      100,
```

I would put this to max 1 cycle
```suggestion
        "number_cycles":                      1,
```